### PR TITLE
docs(linux): Add linux packaging documentation

### PR DESCRIPTION
**Pull request by ermshiperete**
This is an attempt to document Linux packaging of Keyman. Please add comments where you think something is missing or where things are unclear.

I initially thought I would create a page on the wiki for this document and use this PR just for discussion. After looking at the other two files in the `docs` folder it looks like it might fit in there as well. Where do you think this doc should be?

---
This change is [Reviewable](https://reviewable.io/reviews/keymanapp/keyman/2720)

### CODE REVIEW

File: docs/linux-packaging.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
+# Linux packaging
+
+## Package distribution
+
+We use different channels to build and distribute the Linux packages:
+
+- (older version) included in official repos since Ubuntu 19.04 and Debian Buster
+- [Launchpad repo](https://launchpad.net/~keymanapp) for stable and beta versions
```
Comment (darcywong00) [inline docs/linux-packaging.md]: The launchpad repo for beta versions is https://launchpad.net/~keymanapp/+archive/ubuntu/keyman-daily

**Comment by darcywong00** [comment]
Can you move some of the info from 
https://github.com/keymanapp/keyman/tree/master/linux#building-packages and maybe leave a link from that doc to docs/linux-packaging.md ?

File: docs/linux-packaging.md (hunk 1 above)
Comment (ermshiperete) [reply docs/linux-packaging.md]: done

**Comment by ermshiperete** [comment]
> Can you move some of the info from
> https://github.com/keymanapp/keyman/tree/master/linux#building-packages and maybe leave a link from that doc to docs/linux-packaging.md ?

Good idea. I had overlooked that document. Done.

**Pull request merged**